lib/engines/object_store/docs/adr/0001-create-object-store.md

```diff
@@ -0,0 +1,57 @@
+# 3. Build an Object Store in Whitehall to create/edit Content Blocks 
```

Comment:
duplicating/re-wording a bit of [what's in this doc ](https://docs.google.com/document/d/1b-jgRmr-CrtuUhPkgPZbFBt1UG0f8wJ4hOFLTAS00D0/edit)to try and understand it and to narrow the scope to just the creation of the object store...